docs/en-US/component/anchor.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Anchor
+lang: en-US
+---
+
+# Anchor
+
+Through the anchor point, you can quickly find the position of the information content on the current page.
+
+## Basic Usage
+
+The most basic usage
+
+:::demo
+
+anchor/basic
+
+:::
+
+## Horizontal Mode
+
+Horizontally aligned anchors
+
+:::tip
+
+Horizontal Mode does not support sublist slots
+
+:::
+
+:::demo
+
+anchor/horizontal
+
+:::
+
+## Affix Mode
+
+Use the affix component to fix the anchor point within the page.
+
+:::demo
+
+anchor/affix
+
+:::
+
+## Scroll Container
+
+Custom scroll area, use `offset` props can set anchor scroll offset, listen the `link-click` event and prevents browser default behavior then it will not change history.
+
+:::demo
+
+anchor/scroll
+
+:::
+
+## Custom the anchor highlight
+
+Use `get-current-anchor` props to custom the anchor highlight.
+
+:::demo
+
+anchor/getCurrentAnchor
+
+:::
+
+## Anchor link change
+
+Listening for anchor link change
+
+:::demo
+
+anchor/change
+
+:::
+
+## Underline type
+
+set `type="underline"` change to underline type
+
+:::demo
+
+anchor/underline
+
+:::
+
+## Anchor API
+
+### Anchor Attributes
+
+| Property   | Description                              | Type                                                  | Default |
+| ---------- | ---------------------------------------- | ----------------------------------------------------- | ------- |
+| container  | scroll container.                        | `string` \| `HTMLElement` \| `Window `                | —       |
+| offset     | set the offset of the anchor scroll.     | `number`                                              | 0    |
+| bound      | the offset of the element starting to trigger the anchor. | `number`                             | 15      |
+| duration   | set the scroll duration of the container, in milliseconds.          | `number`                   | 300    |
+| marker     | Whether to show the marker.              | ^[boolean]                                            | true   |
+| type       | set Anchor type.                         | ^[enum]`'defalut' \| 'underline'`                     | `default` |
+| get-current-anchor  | customize the anchor highlight.   | ^[Function]`(currentHref: string) => string`          | -   |
```

Comment:
```suggestion
| active | customize the anchor highlight.   | `string` \| ^[Function]`(currentHref: string) => string`          | -   |
```
I think `active` is short, and support the string type. Even I think we can remove this API, it seems useless.

Reply:
This API might be useful. I saw that `antd` only added this after version 3.22.0.
<img width="1394" alt="image" src="https://github.com/element-plus/element-plus/assets/27912232/02542fdd-4fb4-4611-a56a-2964a1fd724f">

Reply:
Can we change this API to `highlight` and support both string and function types?

Reply:
I searched the PR and Issues of antd, `get-current-anchor` is a default active link.
Maybe we can make it like `v-model`? If you have some ideas, you can contact me on WeChat when you are free.